app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -24,14 +24,20 @@ def create
         order.update!(status: :paid, placed_at: DateTime.now, payment_processor_fee_cents: balance_transaction.fee)
         order.events.create(description: "Payment Received")
 
+        # TODO: Does this suffice as a feature flag for the time being?
+        # TODO: Need better conditional abstraction here?
+        if marketplace.square_location_id? && Rails.env.development?
+          order.send_to_square_seller_dashboard(balance_transaction)
+        end
+
         Order::ReceivedMailer.notification(order).deliver_later
         order.events.create(description: "Notifications to Vendor and Distributor Sent")
         Order::PlacedMailer.notification(order).deliver_later
         order.events.create(description: "Notification to Buyer Sent")
 
         SplitJob.perform_later(order: order)
       else
-        raise UnexpectedStripeEventTypeError, event.type
+        # raise UnexpectedStripeEventTypeError, event.type
```

Comment:
This probably should still be there, no?

Reply:
Oops yes.